svm/src/transaction_processor.rs

```diff
@@ -141,8 +140,8 @@ pub struct TransactionProcessingEnvironment<'a> {
     pub blockhash_lamports_per_signature: u64,
     /// The total stake for the current epoch.
     pub epoch_total_stake: Option<u64>,
-    /// The vote accounts for the current epoch.
-    pub epoch_vote_accounts: Option<&'a VoteAccountsHashMap>,
+    /// The stake for vote accounts for the current epoch.
+    pub epoch_vote_stake: HashMap<Pubkey, u64>,
```

Comment:
Can this be a reference rather than an owned value? Avoids cloning outside of the SVM API.